src/components/ForYou/index.tsx

```diff
@@ -159,6 +257,62 @@ const ForYou = () => {
             Let's Begin
           </button>
         </Modal>
+        <Modal isVisible={false} rootClassName="px-[29px] pt-[45px] pb-[30px]">
+          <span className="block text-[20px] font-bold leading-[20px] font-outfit">
+            Select Your Areas of Interest
+          </span>
+          <span className="block mt-[8px] mb-[24px] text-[9px] leading-[10px] font-questrial">
+            Your preferences will help us create a journey unique to you.
+          </span>
+        </Modal>
+
+        <Modal isVisible={false} rootClassName="px-[29px] pt-[45px] pb-[30px]">
+          <span className="block text-[20px] font-bold leading-[20px] font-outfit">
+            Select Your Areas of Interest
+          </span>
+          <span className="block mt-[8px] mb-[24px] text-[9px] leading-[10px]">
+            Your preferences will help us create a journey unique to you.
+          </span>
+
+          <button className="mt-[24px] bg-primary text-white text-[14px] leading-[14px] font-outfit font-bold py-[10px] w-full rounded-[24px] mt-[24px] mb-[16px]">
+            Let's Begin
+          </button>
+        </Modal>
+        <Drawer isVisible={isShowReviews} direction="bottom">
+          <div className="relative w-full flex items-center justify-center py-[17px] px-[19px]">
+            <span className="font-outfit text-[16px] leading-[1] font-bold text-white">
+              Reviews
+            </span>
+            <i
+              className="votigram-icon-cancel text-[14px] text-white absolute top-[15px] right-[18px] cursor-pointer"
+              onClick={() => setIsShowReviews(false)}
+            />
+          </div>
+          <div className="py-[4px]">
+            <List
```

Comment:
what list is this? can you use a better name?

Reply:
Updated